### PR TITLE
Fix webids and space in paths / tags

### PR DESCRIPTION
**Pull request by alexbourret**

### CODE REVIEW

File: python-lib/osisoft_client.py [hunk 1]
```diff
@@ -370,7 +370,7 @@ def get_debug_level(self):
 
     def get_resource_path_params(self, resource_path):
         return {
-            "path": escape(resource_path)
+            "path": resource_path
```
Comment (liamlynch-data) [inline python-lib/osisoft_client.py:373]: We should keep the escape() in and fix those double encoding issues with the function

**Review by liamlynch-data — CHANGES_REQUESTED** [review]
The webid stuff as detailed [here](https://app.shortcut.com/dataiku/story/127323/osisoft-piwebapi-toolbox-not-working-with-webids) is fixed and the code seems fine.

I tested the value handling across all components impacted for regression - I did find something but it was not to do with this change - logged [here](https://app.shortcut.com/dataiku/story/160924/osisoft-attribute-search-does-not-bring-back-values-when-data-type-is-value).

Re the space handling in paths/tags as raised [here](https://app.shortcut.com/dataiku/story/127325/osisoft-piwebapi-toolbox-issue-with-spaces-in-tag-names)  - I had a lot of trouble reproducing the error but investigated the escape function and the impact of removing it. Removing it completely could have some impact as values are not uri encoded later, and there may be values with some of those chars in some systems. It also turns out the escape function is double encoding some things, including spaces - this was probably the original issue. So a better fix is to continue to have the escape function, but fix that issue with it (e.g. by replacing all the chars in one go or doing the `%` first in the loop).

File: python-lib/osisoft_client.py (hunk 1 above)
Comment (alexbourret) [reply python-lib/osisoft_client.py:373]: @liamlynch-data the escape is now fixed and reinstated in [d1523b4](https://github.com/dataiku/dss-plugin-osisoft/pull/35/commits/d1523b47e4f85371606de602081612847015a7a7) and [f064232](https://github.com/dataiku/dss-plugin-osisoft/pull/35/commits/f06423287d8657cf8ca2995ec3bc4363fbebe7be) of the last [PR](https://github.com/dataiku/dss-plugin-osisoft/pull/35).

Comment (liamlynch-data) [reply python-lib/osisoft_client.py:373]: Yep all good - tested and approved in the other PR

**Review by liamlynch-data — APPROVED** [review]
All good, issues highlighted have been resolved and tested in a [later PR in the chain](https://github.com/dataiku/dss-plugin-osisoft/pull/35)

**Pull request merged**